wolfcrypt/src/rsa.c

```diff
@@ -1207,68 +1218,73 @@ static int RsaPrivateDecryptEx(byte* in, word32 inLen, byte* out,
                             enum wc_HashType hash, int mgf,
                             byte* label, word32 labelSz, WC_RNG* rng)
 {
-    int ret = BAD_FUNC_ARG;
+    int ret = RSA_WRONG_TYPE_E;
```

Comment:
Required to be here because of RSA_STATE_DECRYPT_RES, which indicates 'ret' is used uninitialized.